Resources/config/services.xml

```diff
@@ -42,7 +42,7 @@
             <argument type="service" id="sonata.seo.page" />
             <argument type="service" id="translator" />
             <argument type="service" id="cmf_seo.config_values" />
-            <argument type="service" id="cmf_seo.cache" />
```

Comment:
extractors are no longer managed by SeoPresentation, but in a specialized `ExtractorLoader`. Caching was caching which extractors should be executed for which classes, so caching now is the task of this loader.

As the SEO values can be changed by an admin, it doesn't make sense to cache them.